src/chromium/cargo.md

```diff
@@ -35,6 +35,12 @@ Discuss in small groups the policies within your own team and organization,
 and come to a group agreement about what's an acceptable level of risk.
 
 <details>
+Talk about the cultural differences between the `cargo` world and the Chromium
+world: for instance, that code reuse is cheap in `cargo` and that it's
+encouraged to use (and create) lots of small single-purpose crates, which is
+difficult in Chromium (for good reasons). The net effect is that `cargo`
+ecosystem development feels much quicker and more agile.
+
```

Comment:
Looks good, I don't have any blocking concerns.

I wonder if the last sentence could be replaced with something slightly different:

* I agree with that sentence, but I think a comparison of agility is not *needed* here to support the message from this slide.
* I think the more important points are that
    - A big part of the value of adopting Rust comes from tapping into the vibrant, agile Rust community (both for the ecosystem of cargo, and for the Rust language itself) and
    - We are trying to bring these 2 worlds together - not just parashoot-dropping rustc into Chromium, but trying to bring together and integrate 2 different cultures:
        - "Stuff from the internet" (e.g. crates.io libraries which are developed while placing a certain level of trust into rustup, cargo, crates.io) 
        - Chromium stuff (e.g. where builds are hermetic, and most code is produced in-house, goes through a thorough review and audited libraries, and is somewhat monolithic).

Hmmmm... what I wrote above is much longer than a single sentence... :-/  As I said, this is an optional piece of feedback...

Reply:
OK, I decided to take your 137 extra words and change it to -3 words, by removing "much quicker and". I think the gist of your comment is that it's oversimplistic to say that cargo development feels quicker. I think all this nuance is best explained by the teacher's personal experience rather than being explicitly stated in the speaker notes.